spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
##### There are three options for the directive value:

1. the **raw value** with quotes, which preserves character information
2. the **directive value**, which makes it easy to interpret what type of directive it is
3. a **boolean value**, which prevents duplication and node syncing

###### 1. raw value
```json
{ 
 "type": "ExpressionStatement",
 "expression": { 
   "type": "Literal",
    "value": "use strict"
  },
 "directive": "\"use strict\"" 
}
```

###### 2. directive value
```json
{ 
 "type": "ExpressionStatement",
 "expression": { 
   "type": "Literal",
    "value": "use strict"
  },
 "directive": "'use strict" 
}
```

###### 3. boolean value
```json
{ 
 "type": "ExpressionStatement",
 "expression": { 
   "type": "Literal",
    "value": "use strict"
  },
 "directive": true 
}
```

I'd support the second option `directive value`. If  #14 lands, then raw data does seem redundant. Also, If directive values could be different from raw source values, then it seems helpful to flag which directive was matched.